gui/webdisplay/src/TWebWindowWSHandler.hxx

```diff
@@ -26,6 +26,13 @@ namespace Experimental {
 /// just wrapper to deliver websockets call-backs to the TWebWindow class
 
 class TWebWindowWSHandler : public THttpWSHandler {
+
+protected:
+   Bool_t ProcessBatchHolder(std::shared_ptr<THttpCallArg> arg) override
```

Comment:
Do you really need to copy the `shared_ptr` when calling this function? Or would a `const shared_ptr&` be enough?

Reply:
Yes we must. Luckily the caller won't need to change their code, it's basically a bug fix :-)

Reply:
Ok, I change arguments list for this method